MediaBrowser.Api/Playback/TranscodingThrottler.cs

```diff
@@ -48,23 +49,23 @@ private void TimerCallback(object state)
 
             if (options.EnableThrottling && IsThrottleAllowed(_job, options.ThrottleDelaySeconds))
             {
-                PauseTranscoding();
+                await PauseTranscoding();
             }
             else
             {
-                UnpauseTranscoding();
+                await UnpauseTranscoding();
             }
         }
 
-        private void PauseTranscoding()
+        private async Task PauseTranscoding()
         {
             if (!_isPaused)
             {
                 _logger.LogDebug("Sending pause command to ffmpeg");
 
                 try
                 {
-                    _job.Process.StandardInput.Write("c");
+                    await _job.Process.StandardInput.WriteAsync("c");
```

Comment:
mental note: look into `ffmpeg` stdin interface, the way Jellyfin handles throttling now is it pauses transcoding by entering `c` and resumes by writing a newline.